boxit.js

```diff
@@ -0,0 +1,59 @@
+const args = process.argv.slice(2)
+let longest = 0;
+
+for (let i = 0; i < args.length; i++) {
+    if (args[i].length > longest) {
+        longest = args[i].length;
+    }
+}
+
+function drawLine(longest) {
+    let str = ``;
+    for (let i = 0; i < longest; i++) {
+        str += "\u2501";
+    }
+    return str;
+}
+// console.log(drawLine(longest));
```

Comment:
It's best practice clean code wise to remove all **_unnecessary_** console.logs and **_unnecessary_** comments within code